pkg/cgroup/resolve_container_test.go

```diff
@@ -121,3 +122,40 @@ func TestGetAliveContainers(t *testing.T) {
 		})
 	}
 }
+func TestExtractPodContainerIDfromPathWithCgroup(t *testing.T) {
+	tests := []struct {
+		name           string
+		path           string
+		cgroup         int
+		expectedResult string
+	}{
+		{
+			name:           "valid path with cgroup 1",
+			path:           "1:name=systemd:/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-podd0511cd2_29d2_4215_be0f_f77bc0609d99.slice/crio-f93ee491b8ed2680d5a909eb098b14a9430173b57ca1c4efedd8768566d67e8e.scope",
+			cgroup:         1,
+			expectedResult: "f93ee491b8ed2680d5a909eb098b14a9430173b57ca1c4efedd8768566d67e8e",
+		},
+		{
+			name:           "valid path with cgroup 2",
+			path:           "0::/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-pod2c9f8a79_5391_454b_88cb_86190881cb96.slice/crio-a09343ca97901516c25036e2b954421254f8c68b384b536064e8999f0c4ed18d.scope",
+			cgroup:         2,
+			expectedResult: "a09343ca97901516c25036e2b954421254f8c68b384b536064e8999f0c4ed18d",
+		},
+		{
+			name:           "valid path with cgroup v1 with container",
+			path:           "7:cpu,cpuacct:/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-podd0511cd2_29d2_4215_be0f_f77bc0609d99.slice/crio-f93ee491b8ed2680d5a909eb098b14a9430173b57ca1c4efedd8768566d67e8e.scope/container",
+			cgroup:         1,
+			expectedResult: utils.SystemProcessName,
```

Comment:
Why the expected container id is not the `f93ee491b8ed2680d5a909eb098b14a9430173b57ca1c4efedd8768566d67e8e`?

Reply:
because it ends with `/container`